src/lib/Tabs.js

```diff
@@ -0,0 +1,55 @@
+import prefixClass from '../utils/prefixClass'
+
+/**
```

Comment:
This file may be moved into `client/js/lib/statement/` as it is only used within these two templates:

- list_final_group.html.twig
- list_final_group_citizen.html.twig

I'd also suggest to remove the call from `client/js/lib/core/loadLibs.js:20` and move it into 
- client/js/bundles/statement/listFinalGroup.js
- client/js/bundles/statement/listFinalGroupCitizen.js

The `Tabs()` function must then be invoked in the callback of initialize like so:

```js
initialize(components, stores).then(() => {
  Tabs()
})
```

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/71/commits/d90277fdb90ee4349783e5c1e51137121a6e26c6
New PR in demosplan-core: https://github.com/demos-europe/demosplan-core/pull/666